server/controllers/Meetup.js

```diff
@@ -68,13 +68,14 @@ class Meetup {
  * @description RSVP for a meetup
  */
 	static async rsvp(req, res) {
+		const user = req.body.user || 1;
```

Comment:
What happens when const user is 1

Reply:
I have corrected it, used it as a placeholder for running tests but I have set the appropriate parameter as of now.